src/pages/iou/MoneyRequestSelectorPage.js

```diff
@@ -94,6 +95,7 @@ function MoneyRequestSelectorPage(props) {
         <ScreenWrapper
             includeSafeAreaPaddingBottom={false}
             shouldEnableKeyboardAvoidingView={false}
+            shouldEnableMinHeight={DeviceCapabilities.canUseTouchScreen() && props.selectedTab === CONST.TAB.MANUAL}
```

Comment:
But it seems we need this fix only for 'Manual' tab, so the check `&& props.selectedTab === CONST.TAB.MANUAL` make sense.

Reply:
> @sarious Yes agree, but this change has caused this critical bug in IOS, check video in [#30065 (comment)](https://github.com/Expensify/App/pull/30065#issuecomment-1783294762).

it look like it freezes because of emulator problem. But I can get rid of `props.selectedTab === CONST.TAB.MANUAL`, it's not a problem.
Just need to solve the issues with Android emulator, after merging from 'master' and conflicts fixes, it doesn't want to open the app.

Reply:
And I added this fix too.